src/test/resources/topology4.dot

```diff
@@ -94,20 +94,20 @@ digraph G {
 	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000045-\nrepartition" -> "KSTREAM-\nSOURCE-\n0000000049";
 	"KSTREAM-\nREDUCE-\n0000000046" -> "KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000045";
 	"KSTREAM-\nSINK-\n0000000051" -> "snkItemsTopic";
-	"KSTREAM-\nFLATMAP-\n0000000003-\nrepartition" [shape = rect];
-	"KSTREAM-\nFLATMAP-\n0000000005-\nrepartition" [shape = rect];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000025-\nrepartition" [shape = rect];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000014-\nrepartition" [shape = rect];
-	"tblDeptTopic" [shape = rect];
-	"srcFeeTopic" [shape = rect];
-	"srcDeptTopic" [shape = rect];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000045-\nrepartition" [shape = rect];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000009-\nrepartition" [shape = rect];
-	"snkItemsTopic" [shape = rect];
-	"KSTREAM-\nFLATMAP-\n0000000001-\nrepartition" [shape = rect];
-	"srcItemsTopic" [shape = rect];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000014" [shape = cylinder];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000009" [shape = cylinder];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000025" [shape = cylinder];
-	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000045" [shape = cylinder];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000045-\nrepartition" [shape = rect; color = black];
+	"KSTREAM-\nFLATMAP-\n0000000001-\nrepartition" [shape = rect; color = black];
+	"srcItemsTopic" [shape = rect; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000014-\nrepartition" [shape = rect; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000009-\nrepartition" [shape = rect; color = black];
+	"srcDeptTopic" [shape = rect; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000025-\nrepartition" [shape = rect; color = black];
+	"KSTREAM-\nFLATMAP-\n0000000003-\nrepartition" [shape = rect; color = black];
+	"srcFeeTopic" [shape = rect; color = black];
+	"tblDeptTopic" [shape = rect; color = black];
+	"KSTREAM-\nFLATMAP-\n0000000005-\nrepartition" [shape = rect; color = black];
+	"snkItemsTopic" [shape = rect; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000014" [shape = cylinder; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000009" [shape = cylinder; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000025" [shape = cylinder; color = black];
+	"KSTREAM-\nREDUCE-\nSTATE-\nSTORE-\n0000000045" [shape = cylinder; color = black];
```

Comment:
^^ this is due to https://github.com/laserdisc-io/eisner/pull/9/files#diff-69334932c4d29723652ab9b75896e984R4